backend/mora/cli.py

```diff
@@ -395,6 +395,9 @@ def full_run(idp_url, fixture):
     from oio_rest import settings as lora_settings
 
     from tests import util as test_util
+    from tests.test_integration_configuration_settings import Tests as cot
+    ccd = cot._create_conf_data
```

Comment:
Move this function to `tests/util.py` as this is where utility code and test infrastructure belongs.